plugins/source/aws/resources/services/eventbridge/replays.go

```diff
@@ -10,22 +14,57 @@ import (
 func Replays() *schema.Table {
 	tableName := "aws_eventbridge_replays"
 	return &schema.Table{
-		Name:        tableName,
-		Description: `https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Replay.html`,
-		Resolver:    fetchEventbridgeReplays,
-		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "events"),
-		Transform:   transformers.TransformWithStruct(&types.Replay{}),
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_DescribeReplay.html`,
+		Resolver:            fetchReplays,
+		PreResourceResolver: getReplay,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "events"),
+		Transform:           transformers.TransformWithStruct(&eventbridge.DescribeReplayOutput{}, transformers.WithSkipFields("ResultMetadata")),
 		Columns: []schema.Column{
 			client.DefaultAccountIDColumn(false),
 			client.DefaultRegionColumn(false),
 			{
 				Name:     "arn",
 				Type:     schema.TypeString,
-				Resolver: resolveReplayArn,
+				Resolver: schema.PathResolver("ReplayArn"),
 				CreationOptions: schema.ColumnCreationOptions{
 					PrimaryKey: true,
 				},
 			},
 		},
 	}
 }
+
+func fetchReplays(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	var input eventbridge.ListReplaysInput
+	c := meta.(*client.Client)
+	svc := c.Services().Eventbridge
+	for {
+		response, err := svc.ListReplays(ctx, &input)
+		if err != nil {
+			return err
+		}
+		res <- response.Replays
+		if aws.ToString(response.NextToken) == "" {
+			break
+		}
+		input.NextToken = response.NextToken
+	}
+	return nil
+}
+
+func getReplay(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource) error {
+	c := meta.(*client.Client)
+	svc := c.Services().Eventbridge
+
+	replay := resource.Item.(types.Replay)
+
+	out, err := svc.DescribeReplay(ctx, &eventbridge.DescribeReplayInput{
```

Comment:
New code here: We now use a `PreResourceResolver` to get more data including the replay destination.